table/render_init.go

```diff
@@ -255,7 +256,15 @@ func (t *Table) initForRenderSuppressColumns() {
 	shouldSuppressColumn := func(colIdx int) bool {
 		for _, row := range t.rows {
 			if colIdx < len(row) && row[colIdx] != "" {
-				return false
+				// Columns may contain non-printable characters. For example
+				// the text.Direction modifiers. These should not be considered
+				// when deciding to suppress a column.
+				for _, r := range row[colIdx] {
+					if unicode.IsPrint(r) {
```

Comment:
I considered only ignoring the specific characters used by `text.LeftToRight` (`\u202a`) and `text.RightToLeft` (`\u202b`). However, this approach would require the check to be updated if any new modifiers were added later, which could easily be overlooked.

I figure it’s unlikely someone would want to print a column containing only non-printable characters. More likely, this will confuse people—just as it did me when my seemingly empty columns weren't being suppressed. 😅

Reply:
No I like the final approach much better than checking for a couple of characters that we know of today.